dump_img.sh

```
@@ -0,0 +1,25 @@
+echo "Scanning for machine and download when found..."
+echo "Press Ctrl+C to stop scanning."
+
+# Keep scanning until a device is found
+while true; do
+    device=$(find /dev -name "tty.usbmodem*" | head -n 1) # cu.usbmodem
+    
+    # Check if a device was found
+    if [ -n "$device" ]; then
+        echo "Found machine device: $device"
+
+        ~/Library/Arduino15/packages/esp32/tools/esptool_py/5.1.0/esptool --port $device read_flash 0x0 0x800000 la_machine_moods_as_$(date +"%d-%m-%Y").img
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)